charts/adaptive/templates/_helpers.tpl

```diff
@@ -146,6 +155,13 @@ Control plane ports
 }
 {{- end }}
 
+{{/*
+Control plane HTTP endpoint
+*/}}
+{{- define "adaptive.controlPlane.httpEndpoint" -}}
```

Comment:
control plane has in fact 2 http endpoints: 
- the public one 
- the private one (for config and metrics)

Maybe we should better qualify this endpoint ? `publicHttpEndpoint`  ?